test/gsb-test-srv/main.go

```diff
@@ -0,0 +1,468 @@
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/golang/protobuf/proto"
+	gsb_proto "github.com/letsencrypt/boulder/test/gsb-test-srv/proto"
```

Comment:
[Golang style](https://blog.golang.org/package-names) dislikes `_` in package names (or aliases).